atomate/vasp/workflows/presets/core.py

```diff
@@ -281,14 +297,33 @@ def wf_gibbs_free_energy(structure, c=None):
     poisson = c.get("POISSON", 0.25)
     metadata = c.get("METADATA", None)
 
-    # 21 deformed structures
+    # 21 deformed structures: from -10% to +10%
     deformations = [(np.identity(3)*(1+x)).tolist() for x in np.linspace(-0.1, 0.1, 21)]
```

Comment:
These do not deform +/- 10%, but instead +33.1% and -27.% (1.1**3 and 0.9**3) because you are applying a 10% change to each lattice vector